### PR TITLE
add failed node list to web interface

### PR DESCRIPTION
**Pull request by qmlmoon**
implement #390

### CODE REVIEW

**Comment by rmetzger** [comment]
I tested the code on a cluster with a failing job! It works!

**Comment by rmetzger** [comment]
@markus-h: Can you please review the PR ?

File: stratosphere-runtime/resources/web-docs-infoserver/analyze.html [hunk 1]
```diff
@@ -83,6 +83,8 @@ <h3 class="contentbox-header">
          	The job does not have any accumulators
          </div>
       </div>
+
```
Comment (markus-h) [inline stratosphere-runtime/resources/web-docs-infoserver/analyze.html]: Nothing changed here, right? So perhaps we just keep this file out of the pull request.

File: stratosphere-runtime/resources/web-docs-infoserver/js/analyzer.js [hunk 2]
```diff
@@ -58,6 +58,20 @@ function analyzeTime(json, stacked) {
 		$("#run").html(convertTime(job[job.status] - job.SCHEDULED));
 		$("#status").html(job.status);
 		$("#jobtitle").html(job.jobname);
+
+		//create failed table
+		if (job.status == "FAILED") {
+		    failed = "<ul>";
+            $.each(job.failednodes, function(j, failednode) {
```
Comment (markus-h) [inline stratosphere-runtime/resources/web-docs-infoserver/js/analyzer.js]: Can you perhaps change the indentation for better readability?

**Comment by markus-h** [comment]
The code looks good for me, I just commented on two little coding style issues.

**Comment by StephanEwen** [comment]
Merged in f050b708268b781fd86cadd96aeaf6469ebec7e2

**Pull request closed without merging**